flake.nix

```diff
@@ -6,8 +6,14 @@
 
   outputs = { self, nixpkgs, flake-utils }:
     flake-utils.lib.eachDefaultSystem (system: {
-      packages.nixpkgs-review = import ./. {
-        pkgs = nixpkgs.legacyPackages.${system};
+      packages = {
+        nixpkgs-review = import ./. {
+          pkgs = nixpkgs.legacyPackages.${system};
+        };
+        nixpkgs-review-sandbox = import ./. {
```

Comment:
We only should have this on Linux to fix eval.

Reply:
Done, however the code is kinda convoluted.

If you know a better way to do this please say so.